packages/frontend/src/components/ui/SwitchToggle/SwitchToggle.tsx

```diff
@@ -0,0 +1,34 @@
+import { Switch } from '@headlessui/react';
+import clsx from 'clsx';
+
+export interface SwitchProps {
+  ctrlState: boolean;
+  setCtrlState: () => void;
+  srLabel: string;
```

Comment:
@Pearlsky can you add some **jsDoc comments** because from first glance I can't really tell what these props are mean to do - only just make assumptions

Reply:
okay I'd do that